examples/mirror-maker/kafka-mirror-maker-2-custom-replication-policy.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker-2
+spec:
+  version: 2.5.0
```

Comment:
```suggestion
  version: 2.6.0
```

Reply:
Ok I'll do the changes, no need to test with the snapshot, I tried locally. thanks again.

Reply:
Ok, I will then get a proper release of the extensions ready.